resources/manage.rb

```diff
@@ -20,6 +20,7 @@
 def initialize(*args)
   super
   @action = :create
+  @sensitive = true
```

Comment:
I feel it's more appropriate to have this as an attribute with a default value of true as it allows someone to set it to false if for any reason they want the output in their testing environment.